cla_backend/settings/base.py

```diff
@@ -369,6 +369,9 @@
 CONTRACT_2018_ENABLED = os.environ.get("CONTRACT_2018_ENABLED", "True") == "True"
 PING_JSON_KEYS["CONTRACT_2018_ENABLED_key"] = "CONTRACT_2018_ENABLED"
 
+MORTGAGE_CAP_REMOVAL_DATE = os.environ.get("MORTGAGE_CAP_REMOVAL_DATE", "2021-01-28 00:00")
```

Comment:
This works, and I can see the rationale for it (first var matches env var, and we want the env var name to match the final setting name too) but I really don't like it. Maybe hypocritical as I've probably written things like this, but I think I'd much prefer the initial one to have a different name, and probably lowercased in normal Python variable style.
So like
```
mortgage_cap_removal_date = os.environ.get("MORTGAGE_CAP_REMOVAL_DATE", "2021-01-28 00:00")
MORTGAGE_CAP_REMOVAL_DATE = datetime.datetime.strptime(mortgage_cap_removal_date, "%Y-%m-%d %H:%M")
```
or, alternatively, a name that differs by more than just case (maybe with "_string" in?)

Reply:
I agree, you're example seems much more readable